### DISCUSSION TITLE
so-suricata-eve-clean cronjob missing on 2.3.30?

### DISCUSSION THREAD

**Discussion by trocade** [Security-Onion-Solutions/securityonion#3480] [Unsupported Versions] [answered]
I was looking into tuning the `so-suricata-eve-clean` days to keep eve.json logs (as mentioned in the 2.3.30 changelogs)

But this cronjob does not seem to on any of my sensors and when doing a quick find on the manager the only mention of this file is that it exists in this folder:
`/opt/so/saltstack/default/salt/suricata/cron/so-suricata-eve-clean`

I would have expted to find any mention of it in this file:
`/opt/so/saltstack/default/salt/suricata/init.sls`

**Comment by TOoSmOotH** [comment, marked as answer]
Thanks for reporting this! 

https://github.com/Security-Onion-Solutions/securityonion/pull/3516
